### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by sleepeybunney** [sleepeybunney/maelstrom#31] [General]
This discussions thing is new & I don't know what it's like so let's see how it goes.

Things you can post here (if you want):
* Questions
* Feedback
* Feature ideas
* Interesting seeds
* Funny stories/screenshots/clips
* FF8 related garbage

**Comment by xsvnolan** [comment]
Hey love the randomiser thank you so much for this. One thing i thought would be wildly helpful and i'm not sure if you can actually modify spells but if you can modify scan to let us know the common/uncommon/rare drops & mugs it would be very helpful for finding items without having to kill and mug things over n over

**Reply by sleepeybunney** [reply]
Sorry I thought I already replied to this!!! That sounds like a great idea but I don't know how I'd actually do it, so I've noted it down as something to research

**Reply by xsvnolan** [reply]
Hey bunney I've recently started doing some work in the IT field and am curious if/how i could go about helping you in any way.  Would really love to help on the free roam version as i'm quite addicted to it at the moment.

**Reply by sleepeybunney** [reply]
Honestly I have no idea where someone would start with this mess!! But if I think of anything I'll let you know

**Comment by codebibi** [comment]
hello sleepeybunney, i just wanted to report something. i'm french and play on the 2013 version of FF8 and i put the game in english and randomized the game without any problem but when i  start the game it crash after the squarsoft logo and say that an unknown error occur. it looks awesome and really fun to play, hope it will be fix so i could play it one day.

**Reply by sleepeybunney** [reply]
Try deleting everything in the FF8 install folder & then run "verify game files" from your steam library. See if you can start the game without randomising it first. Let me know how it goes!

**Reply by codebibi** [reply]
i literally deleted everything in the FF8 folder and verify game files and know it works i am so glad. by the way i also have a little question. after completely closing the game, if i launch the game will it still be randomized or do i have to randomize it again ? sorry if my question sound stupid but i had to restart the game for something and it didn't seemed randomized anymore after that.
also sorry for the delayed response i didn't saw you respond to me.

**Reply by sleepeybunney** [reply]
Yes it patches the game files so it'll stay randomised until you reroll or wipe everything out again

**Comment by Max33** [comment]
Hey sleepeybunney, it's a really work you did, I enjoyed making randomized runs.
As a French, I wanted to know what do you mean by "On Hold" for the #21 non-english language support for the 2013 Steam version ?
Related: https://github.com/sleepeybunney/maelstrom/projects/1

Thank you!

**Reply by sleepeybunney** [reply]
It means I started working on it but I didn't finish it

**Reply by Max33** [reply]
Thanks! :)

**Comment by codebibi** [comment]
is there a boss spawning from killing tonberrys 'cause i've been killing them for a while and nothing spawn. and i've been wandering about jumbo cactuar too, is it still on this island even after killling it earlier in the game and having the gf already ?

**Reply by sleepeybunney** [reply]
Jumbo Cactuar won't spawn if you already have the GF - there's a bug open to fix it so you don't get locked out of that boss.

The Tonberry boss just normally takes forever to spawn!

**Comment by NecroCorey** [comment]
Hey I was curious if there were any plans to make a discord to discuss this? Git isn't terribly user friendly for most people. It would be a cool way to post updates and get a broader range of input from people.

Anyway thanks for all the hard work. The rando looks awesome. I only know a bit of Python so I can't help, but I'm cheering for you. :D

**Reply by sleepeybunney** [reply]
Thanks! I've thought about setting up a discord but idk, it seems like a lot to manage. Maybe at some point

**Comment by roxas1511** [comment]
I want to try it out, and im about to buy ff8 on my pc to do it but i was trying to figure out how well does it work with the remastered version?
I suppose I could get the og one on steam but I heard it doesnt have the times 3 speed

**Reply by sleepeybunney** [reply]
I'd recommend the OG Steam version because it's cheaper, it includes the Chocobo World minigame, and it has one extra option for randomisation in Maelstrom. But the difference isn't huge, you can't really go wrong. Both versions have issues you'll probably need to fix (the music on the regular version, the framerate on Remastered). Pretty sure they have the same cheats & stuff.

**Comment by roxas1511** [comment]
I made a discord for this MMO game I play, they can be a lot to manage, but id say it mostly be people asking for more stuff or why stuff isn’t working lol. Thank you, I’m going to buy it today or tomorrow. Have you had a lot of new comers with Primal’s vids or stay  pretty steady anyways?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by sleepeybunney** [reply]
Yeah those videos sent a lot of people over

**Comment by roxas1511** [comment]
Hey do you know of any ff8 mods to keep Seifer in party?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by roxas1511** [comment]
Gave in and bought the steam remastered version. Can’t seem to get Maelstrom to work everyime I click go it  asys saving and then closes itself out.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by codebibi** [comment]
very happy to be able to play the game in french but the cut spells doesn't have a name, it is just blank.

**Reply by codebibi** [reply]
i am currently playing on lastest version of the randomizer on the seed 588979399 and i can't finish it because i have to fight gargantua twice in the ragnarok and everytime the second fight makes my game crash.

**Reply by sleepeybunney** [reply]
Thanks for letting me know, I'll fix this in the next update. If you still want to complete this game you could change the seed for this section and then change it back afterwards.

**Comment by JayH2971** [comment]
Hi, I wanted to make a suggestion for your planned feature:

> More substantial rebalancing for shuffled bosses, so they are generally beatable at the point they are encountered

An idea I had is to change the boss pool so they can only come from fixed discs in the game.

In disc 1: bosses from discs 1 & 2
In disc 2: bosses from discs 1, 2, & 3
In disc 3: bosses from discs 2, 3, & 4
In disc 4: bosses from discs 3 & 4

Tonberry King would be counted as a disc 3 boss, and Diablos + Brothers as disc 1 bosses, due to their placement in the GF list.

This could be an optional setting that allows for a more race-friendly environment, rather than getting stuck with Catoblepas at the Fire Cavern, Trauma at Dollet, Adel replacing X-ATM, etc. There would still be room for surprise and strategy, but no crazy jumps in power.

EDIT: Perhaps the Propagators could be on any disc, since they're kind of a bottleneck in terms of boss spacing, and they're not powerful enough to be real bosses.

**Reply by JayH2971** [reply]
Also, what say Omega Weapon becomes its own option, to mix into the pool or not? I just got it replacing the Purple Propagators, meaning I'd have to defeat it _twice_, with just Squall and Rinoa in the middle of Disc 3. That's a deal breaker!

**Comment by rovingmystic** [comment]
I have a suggestion.
I would like an option to guarantee GFs have the Boost ability.  It kinda sucks when none have it most of the time.  Especially Eden, that can easily double its damage output
